gpflow/models/sgpr.py

```diff
@@ -26,37 +26,68 @@
 from ..mean_functions import Zero, MeanFunction
 
 
-class SGPRUpperMixin(GPModel):
+class SGPRBase(GPModel):
+    """
+    Common base class for SGPR and GPRFITC that provides the common __init__
+    and upper_bound() methods.
```

Comment:
if it's going to remain public, you may want to change this docstring to reflect that

Reply:
perhaps describe what it does rather than how it's used?